src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -84,6 +84,27 @@ class ClaimSimulation extends Simulation {
     })
   )
 
+  val randomNhsNos = Iterator.continually(
+    // Random number will be accessible in session under variable "OrderRef"
+    Map("nhsno" -> {
+      randomAlphaNumericString(5)
```

Comment:
Should this not be 10 instead of 5, as nhs number is 10 length

Reply:
@adeel718 the validation which dean developed on test/nhs-number page allows only 5 characters, more than 5 characters is invalid